doc/newsbeuter.txt

```diff
@@ -662,6 +663,34 @@ In order to use this mapping, you need to configure the flags that shall be used
 After that, use these flags when you edit flags for an article, and these articles
 will be starred resp. published.
 
+ownCloud News Synchronization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For use the ownCloud News support you need to configure a few things
+
+First, set your urls-source and also set where the ownCloud instance reside:
+
+	urls-source "ocnews"
+	ocnews-url "https://localhost/owncloud"
+
+In addition, it requires username and password for authentication:
+
+	ocnews-login "user"
+	ocnews-password "password"
+
+ownCloud News API uses HTTP basic auth, therefore running ownCloud with SSL is highly
+recommended.
+
+In order to use this mapping, you need to configure the flags that shall be used:
```

Comment:
It's not clear which mapping you mean here. Can you rephrase that? It's okay to repeat things you wrote in `configcommands.dsv`—we don't expect users to read the docs in their entirety before even starting the program.